test/data.jl

```diff
@@ -51,7 +51,9 @@
     # kwargs
     sdata = georef((z=rand(100),))
     h = EmpiricalHistogram(sdata, :z; nbins=10)
-    @test length(h.weights) == 10
+    c, w = values(h)
+    @test length(c) == 10
+    @test length(w) == 10
```

Comment:
I think we need to move this test to the new histograms.jl test file. Can you do that as well? I forgot to fix it in my latest commit.

Reply:
Sure

Reply:
I mean move all tests related to EmpiricalHistogram to the new histograms.jl test file. I think this test/data.jl may contain other instance of EmpiricalHistogram that should be moved as well.